contractcourt/htlc_outgoing_contest_resolver.go

```diff
@@ -62,7 +62,10 @@ func (h *htlcOutgoingContestResolver) Resolve(
 	//
 	// TODO(roasbeef): Implement resolving HTLCs with custom records
 	// (follow-up PR).
-	if len(h.htlc.CustomRecords) != 0 {
+	if len(h.htlc.CustomRecords) != 0 && h.isTapscriptRoot {
```

Comment:
I believe we could _just_ check `h.isTapscriptRoot` here as our check? 

Happy to remove the `CustomRecords` check if it's useful, but since this is a to-be-removed workaround I didn't think it matters that much (and more specific = more good?).